package.json

```diff
@@ -21,13 +21,12 @@
   "author": "",
   "license": "Apache-2.0",
   "devDependencies": {
-    "@code-dot-org/p5.play": "1.3.4-cdo",
+    "@code-dot-org/p5.play": "1.3.5-cdo",
```

Comment:
Need to add a `@code-dot-org/p5` dev dependency?

Reply:
I had one, but removed it because it was implied as a sub-dependency of p5.play. If I put one here, I'll need to keep it in sync with the version that p5.play is pulling in.